.travis.yml

```diff
@@ -1,11 +1,26 @@
 language: r
-r:
-  - oldrel
-  - release
-  - devel
+matrix:
+  include:
+    - name: "Roxygen check"
+      r: release
+      r_packages:
+        - devtools
+        - roxygen2
+      script: ./tools/checkDocsCurrent.sh
```

Comment:
One option would be to put this `before_script`, then you could run this along with the current release check, which would save on a build. The other option would be to do this as a separate job up front before any of the R specific version checks are run...

Reply:
Our concern was that doing this in-line with the R tests would pollute our tests with a bunch of other packages. I originally had it at the end of our tests, but it felt silly to do it three times redundantly -- especially now that we have a test for yarn/JS stuff in there, too.

Let me know if you think I'm missing something or if there's a cleaner way to flesh out this matrix.